shared/actions/wallets.js

```diff
@@ -107,9 +107,9 @@ const requestPayment = (state: TypedState) =>
     {
       amount: state.wallets.building.amount,
       // FIXME -- support other assets.
-      asset: emptyAsset,
+      asset: state.wallets.building.currency ? null : emptyAsset,
       recipient: state.wallets.building.to,
-      // TODO -- support currency
+      currency: state.wallets.building.currency || undefined,
```

Comment:
@buoyad Is this what you meant? (I didn't touch the send path, since Steve seems to be working on that)